SSDynamicText/SSDynamicLabel.m

```diff
@@ -67,14 +49,33 @@ - (void)dealloc {
     [self ss_stopObservingTextSizeChanges];
 }
 
+#pragma mark - Accessors
+
+- (void)setDefaultFontDescriptor:(UIFontDescriptor *)defaultFontDescriptor {
+    self.textSizeChanger.defaultFontDescriptor = defaultFontDescriptor;
+    super.defaultFontDescriptor = defaultFontDescriptor;
+}
+
+- (void)setFont:(UIFont *)font {
+    super.font = font;
+    [self setupDefaultFontDescriptorBasedOnFont:self.font];
+}
+
 #pragma mark - Private Methods
 
+- (SSDynamicTextSizeChanger *)textSizeChanger {
+    if (_textSizeChanger == nil) {
+        _textSizeChanger = [self createTextChanger];
+    }
+    return _textSizeChanger;
+}
+
 - (SSDynamicTextSizeChanger *)createTextChanger {
     SSDynamicTextSizeChanger *changer = [[SSDynamicTextSizeChanger alloc] init];
     __weak typeof(self) weakSelf = self;
 
     changer.fontChangeBlock = ^(UIFont *font) {
-        [super setFont:font];
+        super.font = font;
```

Comment:
A minor stylistic thing, but this is one area where I don't tend to use dot syntax. `[super setFont:font]` feels a bit more explicit about which selector we're invoking.